### PR TITLE
Add fingerprint for EU 2019 Civic Hatch

### PR DESCRIPTION
**Pull request by alancyau**
Please close/delete PR #2695 and use this PR instead.

dongle ID: 88628232a4259218

This is for an EU 2019 Honda Civic hatchback, which was probably why the US Civic fingerprints didn't work.

Thanks to Discord user dave237.

### CODE REVIEW

File: selfdrive/car/honda/values.py [hunk 1]
```diff
@@ -598,6 +602,7 @@ class CAR:
       b'46114-TLA-A050\x00\x00',
       b'46114-TLA-A930\x00\x00',
       b'46114-TMC-U020\x00\x00',
+      b'39494-TGL-G030\x00\x00',
```
Comment (VirtuallyChris) [inline selfdrive/car/honda/values.py:605]: Revert this change. On some cars, we don't include all the possible fw queries, and you put this one is in the Civic_diesel

**Review by VirtuallyChris — CHANGES_REQUESTED** [review]

File: selfdrive/car/honda/values.py (hunk 1 above)
Comment (alancyau) [reply selfdrive/car/honda/values.py]: Removed `39494-TGL-G030\x00\x00` from the `electricBrakeBooster`.

File: selfdrive/car/honda/values.py [hunk 2]
```diff
@@ -597,7 +600,7 @@ class CAR:
       b'46114-TLA-A040\x00\x00',
       b'46114-TLA-A050\x00\x00',
       b'46114-TLA-A930\x00\x00',
-      b'46114-TMC-U020\x00\x00',
+      b'46114-TMC-U020\x00\x00',
```
Comment (VirtuallyChris) [inline selfdrive/car/honda/values.py:603]: Just remove this extra space and we're good to go!

**Review by VirtuallyChris — CHANGES_REQUESTED** [review]

File: selfdrive/car/honda/values.py (hunk 2 above)
Comment (alancyau) [reply selfdrive/car/honda/values.py]: Good catch! Extra space removed.
